### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by Steve-xmh** [Steve-xmh/scl#92] [Announcements]
- 增加 NeoForge 模组加载器的安装和识别支持
- 将离线玩家 UUID 从随机生成变更为 Bukkit 方式生成
- 特别提供了无日志构建版本，体积更小，便于携带哦！
- 重新使用 async-h1 作为 HTTP 请求框架，减小体积的同时也降低了性能（雾）
- 优化/修复/新增了一些问题/特性

<hr /><em>This discussion was created from the release <a href='https://github.com/Steve-xmh/scl/releases/tag/v1.2.0'>v1.2.0</a>.</em>

**Comment by Twor** [comment]
+ os: Linux 6.8.2  x86_64
+ pkg: SharpCraftLauncher-20240216-1.2.0-linux-x86_64.tar.gz

无法选择目录，没有确定或者选择按钮

![image](https://github.com/Steve-xmh/scl/assets/18731946/7e575415-b07b-4787-99dc-a8efd8fb4d9b)
